content/java/concepts/calendar/terms/getCalendarType/getCalendarType.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'getCalendarType()'
+Description: 'Retrieves the type or format of the calendar represented by the Calendar object, returns a string representing the calendar type'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Classes'
+  - 'Attributes'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **getCalendarType()** method is available in the Calendar class in Java. It is used to retrieve the type or format of the calendar represented by the Calendar object. This method returns a string representing the calendar type, such as "gregory", "japanese", "buddhist", etc. This method does not accept any argument as a parameter.
+
+## Syntax
+
+```java
+public String getCalendarType()
+```
+
+## Example
+
```

Comment:
There should be a brief introductory sentence before the code block

Reply:
Added short description. @caupolicandiaz I hope I've made all the changes.